settings.js

```diff
@@ -242,7 +242,7 @@ function keystrToKeycombo(keystr) {
         keystr = keystr.replace('Above_Tab', 'a');
         aboveTab = true;
     }
-    let [ok, key, mask] = Gtk.accelerator_parse(keystr);
+    let [key, mask] = Gtk.accelerator_parse(keystr);
```

Comment:
Thanks @Lythenas!

This should be the only change (revert) needed to fix gnome 43, 42 keybind regressions.